content/concepts/networks-and-servers/amendments.md

```diff
@@ -62,6 +62,12 @@ The voting configuration of a `rippled` server has no impact on it becoming amen
 
 You can unblock amendment blocked servers by upgrading to the newest version of `rippled`.
 
+### Amendment Blocked in Clio Servers
+<a id="amendment-blocked-clio"></a>
+
+In Clio, being amendment blocked means that Clio encountered an unknown field type during its ETL process.
+This can happen because Clio was built with a version of `libxrpl` that does not know the field in question.
+In order to unblock Clio one have to upgrade to a version that was built with a compatible `libxrpl`.
```

Comment:
Putting the sentences on separate lines like this doesn't really do anything. Markdown treats them as hard-wrapped continuations of the same paragraph.

Also, "ETL" is jargon that some people may not know, and `libxrpl` is not widely known either, so this is likely to be confusing to new users. We can rephrase to avoid "ETL" and provide just a little more context about what `libxrpl` is.

Lastly, you can make the last sentence simpler. "In order to" can almost always be changed to "To", and using the imperative is clearer than saying "one have to upgrade".

In total, here's my suggestion:

```suggestion
The Clio server can become amendment blocked if it encounters an unknown field type while loading ledger data. This occurs if the field is newer than the `libxrpl` dependency that was used when building Clio. To unblock your Clio server, upgrade to a newer Clio release that was built with a compatible `libxrpl`.
```